src/dirtyfields/compare.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+import pytz
+import warnings
+
+from django.conf import settings
+from django.utils import timezone
+
+
+def raw_compare(cls, new_value, old_value):
+    return new_value != old_value
```

Comment:
For me its a bit unusual that the compare function returns true in case of inequality.